scripts/generate_cves_25_10/find_severe_cves.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import csv
+import logging
+import sys
+
+# For the purposes of this script, severe CVES are considered CVEs that are
+# (KEV or High or Critical) and unfixable
+
+LOG_FORMAT = "%(levelname)s:%(name)s: %(message)s"
+logging.basicConfig(level=logging.INFO, stream=sys.stdout, format=LOG_FORMAT)
+logger = logging.getLogger(__name__)
+
+OUTPUT_CSV_FILE = "severe_cves.csv"
+
+
+def find_critical_cves(input_csv_file, output_csv_file):
+    """Open a CSV file and produce a new one with all severe vulenrabilities."""
+    critical_counter = 0
+    critical_unfixed_counter = 0
+    critical_unfixed_rows = []
+
+    critical_images = set()
+
+    with open(input_csv_file, "r", encoding="utf-8") as file:
+        reader = csv.DictReader(file)
+        headers = reader.fieldnames
+        for row in reader:
+            if (
+                row.get("Is KEV?") == "Yes"
+                or row.get("Severity") == "High"
+                or row.get("Severity") == "Critical"
+            ):
+                critical_counter += 1
+
+                if row.get("Can it be remediated?") == "No":
+                    critical_unfixed_counter += 1
+                    critical_unfixed_rows.append(row)
+
+                    components = [
+                        comp.strip()
+                        for comp in row.get("Affected Component").split(",")
+                    ]
+                    for component in components:
+                        critical_images.add(component)
```

Comment:
question: why collect `critical_images` if we do not include them in the output? we just want the counter right?